src/app/app.component.html

```diff
@@ -172,11 +171,11 @@ <h3 class="md-display-1">{{ ::$ctrl.data.sections.sponsors.title }}</h3>
       <p ng-repeat="content in ::$ctrl.data.sections.sponsors.content">
         {{ ::content }}
       </p>
-      <div layout-xs="column" layout="row" layout-align="center center" class="sponsors">
+      <div layout-xs="column" layout="column" layout-align="center center" class="sponsors">
```

Comment:
this is reduntant, if you wanna have always column flex layout just remove `layout-xs`